packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -123,10 +123,32 @@ export interface IGarbageCollectionSnapshotData {
 	deletedNodes: string[] | undefined;
 }
 
+/**
+ * @experimental - Can be deleted/changed at any time
+ * Contains the necessary information to allow DDSes to do incremental summaries
+ */
+export interface IIncrementalSummaryContext {
+	/**
+	 * The sequence number of the summary generated that will be sent to the server.
+	 */
+	wipSummarySequenceNumber: number;
+	/**
+	 * The sequence number of the most recent summary that was acknowledged by the server.
+	 */
+	lastAckedSummarySequenceNumber: number;
```

Comment:
nit: `latestSummarySequenceNumber` to keep in sync with the naming in summarizer node.